resolveImportsCached.mjs

```diff
@@ -1,16 +1,24 @@
+// @ts-check
+
 import resolveImports from "./resolveImports.mjs";
 
 const cache = new Map();
 
+/**
+ * Resolves the imports for a given module and caches the result.
+ * @param {string} module The path to the module.
+ * @returns An array containing paths to modules that can be preloaded, or otherwise `undefined`.
+ */
 export default async function resolveImportsCached(module) {
+  /** @type {Array<string> | undefined} */
```

Comment:
The cache itself should be typed, instead of everywhere it's used.

Reply:
Oh that makes way more sense.

Reply:
Sorted.